yazi-plugin/preset/components/header.lua

```diff
@@ -46,7 +51,7 @@ function Header:flags()
 	if finder then
 		t[#t + 1] = string.format("find: %s", finder)
 	end
-	return #t == 0 and "" or " (" .. table.concat(t, ", ") .. ")"
+	return #t == 0 and "" or "(" .. table.concat(t, ", ") .. ")"
```

Comment:
Please keep this unchanged

Reply:
Here I removed the space to fix this:
![image](https://github.com/user-attachments/assets/08dee9cd-5495-43ce-9460-3c07ea5ecf10)